pkgs/os-specific/linux/displaylink/default.nix

```diff
@@ -28,28 +28,28 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "displaylink";
-  version = "6.0.0-24";
+  version = "6.1.0-17";
 
   src = requireFile rec {
-    name = "displaylink-600.zip";
-    sha256 = "1ixrklwk67w25cy77n7l0pq6j9i4bp4lkdr30kp1jsmyz8daaypw";
+    name = "displaylink-610.zip";
+    sha256 = "sha256-RJgVrX+Y8Nvz106Xh+W9N9uRLC2VO00fBJeS8vs7fKw=";
```

Comment:
Should `sha256` be updated to `hash`?

Reply:
I dont know if `requiredFile` works with `sha256`. Try it and let us know?

Reply:
`requiredFile` supports both the `sha256` and the `hash` arguments, the current PR works, but there's been a movement towards reducing the use of the `sha256` argument in favor of `hash` in many fetchers, so I just wondered if it would fit in this PR.